tests/Unit/Helpers/PointwiseFunctions/AnalyticData/TestHelpers.hpp

```diff
@@ -23,9 +23,10 @@ void test_tag_retrieval(const Solution& solution, const Coords& coords,
   tmpl::for_each<TagsList>([&](const auto tag_v) {
     using tag = tmpl::type_from<decltype(tag_v)>;
     const auto single_var = solution.variables(coords, tmpl::list<tag>{});
-    CHECK(tuples::get<tag>(single_var) == tuples::get<tag>(vars_from_all_tags));
-    CHECK(tuples::get<tag>(single_var) ==
-          tuples::get<tag>(vars_from_all_tags_reversed));
+    CHECK_ITERABLE_APPROX(tuples::get<tag>(single_var),
+                          tuples::get<tag>(vars_from_all_tags));
```

Comment:
test_tag_retrieval was giving an error when using CHECK.

Reply:
1. There are no other changes in this commit, so probably this should be in one of the later commits.
2. Do you understand why?  It might be fine, but I don't see an obvious reason for them to differ,

Reply:
In offline discussion we decided this change is fine squashed into a later commit.  It appears to be optimization related, and this class has more interaction between fetched variables than most of the others so that's not terribly unexpected.